src/test/java/net/minecraftforge/debug/client/AudioStreamTest.java

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
Please rerun the license task - we use the SPDX format now.

Reply:
updated license